lib/schema.js

```diff
@@ -274,6 +274,13 @@ internals.method = Joi.object({
 });
 
 
+internals.methodObject = Joi.object({
+    name: Joi.string(),
```

Comment:
Yes. At the moment that's still taken care of here: 

https://github.com/hapijs/hapi/blob/master/lib/methods.js#L42-L43

But I can add it to this schema too to be more explicit about it.